Runtime/CesiumTileExcluder.cs

```diff
@@ -0,0 +1,28 @@
+﻿using UnityEngine;
+
+namespace CesiumForUnity
+{
+    [ExecuteInEditMode]
+    public abstract partial class CesiumTileExcluder : MonoBehaviour
```

Comment:
Could use some brief documentation echoing what we say in the change log.

Reply:
Done